mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -156,6 +195,7 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 start
 		ARG 1 end
 		ARG 2 delta
+	METHOD method_22860 fwrapDegrees (D)F
```

Comment:
May be `fWrapDegrees` or `wrapDegreesF` (note your casing)

Reply:
I was following the pattern of `lfloor`, it differs from `wrapDegrees(D)D` only in return type. Also, this is a deprecated method that people should probably use the other methods in favour of.

Reply:
Not really, we have `nextInt` `nextDouble`, so `floorLong` etc should be more accurate and match the pattern better